test/unit/options.js

```diff
@@ -119,3 +121,55 @@ test('useEntryKeys, exclude sourcemap', async (t) => {
 
   t.snapshot(manifest);
 });
+
+test('processAssetsStage', async (t) => {
+  const FIRST_PROCESS_ASSETS_STAGE = 0;
+  const SECOND_PROCESS_ASSETS_STAGE = 1;
+  let assets;
+
+  class LastStagePlugin {
+    /* eslint-disable class-methods-use-this */
```

Comment:
Let me know if I should adjust this. I disabled the rule because this is a canonical example of a plugin, but I'm happy to futz with it if you'd prefer the rule not be turned off here.

Reply:
all good. tests frequently need rules adjusted